index.html

```diff
@@ -2406,6 +2634,1425 @@ <h2>
+              </dd>
+              <dt>
+                strongMagnitude
+              </dt>
+              <dd>
+                is the intensity level for low-frequency vibration.
+              </dd>
+              <dt>
+                weakMagnitude
+              </dt>
+              <dd>
+                is the intensity level for high-frequency vibration.
+              </dd>
+              <dt>
+                leftTrigger
+              </dt>
+              <dd>
+                is the intensity level for vibration localized to the bottom left front button.
+              </dd>
+              <dt>
+                rightTrigger
+              </dt>
+              <dd>
+                is the intensity level for vibration localized to the bottom right front button.
+              </dd>
+            </dl>
+          </section>
+        </section>
+        <section>
+          <h4>
+            Commands
+          </h4>
+          <section>
+            <h5>
+              The emulation.simulateDevice Command
+            </h5>
+            <pre class="cddl">
+              emulation.SimulateDevice = {
+                method: "emulation.simulateDevice",
+                params: emulation.SimulateGamepadParams,
+              }
+              Returns: deviceId: text
+            </pre>
+            <p>
+              The [=remote end steps=] with command parameters |params| are:
+            </p>
+            <ol>
+              <li>
+                Let |source| be the result of [=create a gamepad input source=] with |params|.
+              </li>
+              <li>
+                Let |simulatedGamepads| be the [=simulated gamepad map=].
+              </li>
+              <li data-cite="webcrypto">
+                Let |deviceId:string| be the result of calling [=generate a random UUID=].
+              </li>
+              <li>
+                [=map/Set=] |simulatedGamepads|[|deviceId|] to |source|.
+              </li>
+              <li>
+                Run the steps [=when a gamepad becomes available on the system=] for |gamepad|.
+              </li>
+              <li>
+                Return |deviceId|.
+              </li>
+            </ol>
+          </section>
+          <section>
+            <h5>
+              The emulation.removeSimulatedDevice Command
+            </h5>
+            <pre class="cddl">
+              emulation.RemoveSimulatedDevice = {
+                method: "emulation.removeSimulatedDevice",
+                params: emulation.RemoveSimulatedDeviceParams,
+              }
+              Returns: void
+            </pre>
+            <p>
+              The [=remote end steps=] with command parameters |params| are:
+            </p>
+            <ol>
+              <li>
+                Let |simulatedGamepads| be the [=simulated gamepad map=].
+              </li>
+              <li>
+                If |simulatedGamepads|[|params|.|deviceId|] does not [=map/exist=], abort these steps.
```

Comment:
1. We should return a error if there is no such an simulated device.
2. `params` is a map, so `|params|["<code>deviceId</code>"]` is the correct way to access it's values by keys.
```suggestion
                If |simulatedGamepads|[|params|["<code>deviceId</code>"]] does not [=map/exist=], return [=error=]
                with [=error code=] [=invalid argument=].
```